api/README.md

```diff
@@ -21,4 +39,144 @@ enum Foo {
   MY_VALUE = 2;
   MY_OTHER_VALUE = 3;
 }
-```
\ No newline at end of file
+```
+
+## Rules
+
+Below are conventions followed in the Clutch APIs that are unfortunately not automatically linted yet.
+
+The rules are meant to take as much guesswork out of naming things as possible by making the filename, package name, and services match. HTTP annotations are literal RPC, instead of trying to map RESTful concepts back onto the API.
+
+### All HTTP annotations should use the POST method
+
+**Examples of incorrect definition for this rule:**
+```
+rpc GetThing(GetThingRequest) returns (GetThingResponse) {
+  option (google.api.http) = {
+    get : "/v1/foo/thing"
+  };
+}
+```
+- :x: HTTP annotation uses GET instead of POST.
+
+**Examples of correct definition for this rule:**
+```
+rpc GetThing(GetThingRequest) returns (GetThingResponse) {
+  option (google.api.http) = {
+    post : "/v1/foo/getThing"
+    body : "*"
+  };
+}
+```
+
+### Version should always be the trailing component of package
+
+**Examples of incorrect definition for this rule:**
+```proto
+package clutch.aws.v1.ec2;
+```
+
+- :x: Last component of package is `ec2`, not `v1`.
+
+```proto
+package clutch.v1.healthcheck;
+```
+
+- :x: Last component of package is `healthcheck`, not `v1`.
+
+**Examples of correct definition for this rule:**
+```proto
+package clutch.healthcheck.v1;
+```
+
+```proto
+package clutch.aws.ec2.v1;
+```
+
```

Comment:
nit: perhaps swap the order of the correct definitions so that it matches the order of the incorrect definitions?